docs/exposing-binding-data/olm-descriptors.md

```diff
@@ -0,0 +1,182 @@
+---
+sidebar_position: 5
+---
+
+# Adding Binding Metadata as OLM Descriptors
+
+If your operator is distributed as an [Operator
+Bundle](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md),
+you can add OLM descriptors to expose the binding data.  The OLM descriptors are
```

Comment:
> you can add OLM descriptors to expose the binding data

you can add OLM descriptions to describe what binding data are exposed. Can we put some reference to OLM descriptors from OLM docs?